source/@types/drupal/index.d.ts

```diff
@@ -0,0 +1,66 @@
+declare module "drupal" {
+  interface GessoSettings {
+    gesso: {
+      backToTopThreshold?: number;
+      backToTopSmoothScroll?: boolean;
+    };
+  }
+
+  type Settings = object & GessoSettings;
+
+  interface Behavior {
+    attach?: (context: HTMLElement | Document, settings: Settings) => void;
+
+    detach?: (
+      context: HTMLElement | Document,
+      settings: Settings,
+      trigger: string
+    ) => void;
+  }
+
+  const Drupal: {
+    attachBehaviors(context?: HTMLElement | Document, settings?: Settings): void;
+
+    throwError(error: Error | string): void;
+
+    checkPlain(str: string): string;
+
+    formatString(str: string, args: object): string;
+
+    stringReplace(str: string, args: object, keys?: any): string;
+
+    t(str: string, args?: any, options?: string): string;
+
+    formatPlural(
+      count: number,
+      singular: string,
+      plural: string,
+      args?: object,
+      options?: object
+    ): string;
+
+    encodePath(item: string): string;
+
+    url: {
+      (path: string): string;
+
+      toAbsolute(url: string): string;
+
+      isLocal(url: string): boolean;
+    };
+
+    theme: {
+      (func: string, options: any): string | HTMLElement;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/forumone/gesso/pullRequest?prid=8821041)
```suggestion
      (func: string, options: unknown): string | HTMLElement;
```